Paystack/UI/PSTCKAuthViewController.m

```diff
@@ -42,8 +42,15 @@ - (void)viewDidLoad {
     
     UIBarButtonItem *cancelButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemCancel target:self action:@selector(tappedCancelButton:)];
     self.navigationItem.leftBarButtonItem = cancelButton;
+    // Adds javascript to make content width the device width.
+    NSString *jScript = @"var meta = document.createElement('meta'); meta.setAttribute('name', 'viewport'); meta.setAttribute('content', 'width=device-width'); document.getElementsByTagName('head')[0].appendChild(meta);";
+    WKUserScript *wkUScript = [[WKUserScript alloc] initWithSource:jScript injectionTime:WKUserScriptInjectionTimeAtDocumentEnd forMainFrameOnly:YES];
+    WKUserContentController *wkUController = [[WKUserContentController alloc] init];
+    [wkUController addUserScript:wkUScript];
+    WKWebViewConfiguration *wkWebConfig = [[WKWebViewConfiguration alloc] init];
+    wkWebConfig.userContentController = wkUController;
```

Comment:
Okay, got it. Figured out how to produce the same results with WKWebView is to use Javascript to modify the content width. It would be nice to test with the actual 3DS pages that are "tiny".